aws/resource_aws_iam_role_test.go

```diff
@@ -261,6 +295,39 @@ func testAccAddAwsIAMRolePolicy(n string) resource.TestCheckFunc {
 	}
 }
 
+func testMaxSessionDuration(rName string) string {
```

Comment:
The signature here should be:
```go
func testAccCheckIAMRoleConfig_MaxSessionDuration(rName string, maxSessionDuration int) string {
```

Then accept the parameter in the HCL configuration:
```hcl
  max_session_duration = %d
```

So we can remove the other two test configurations.